reference/5.1/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -19,24 +19,29 @@ Starts one or more processes on the local computer.
 
 ```
 Start-Process [-FilePath] <String> [[-ArgumentList] <String[]>] [-Credential <PSCredential>]
- [-WorkingDirectory <String>] [-LoadUserProfile] [-NoNewWindow] [-PassThru] [-RedirectStandardError <String>]
- [-RedirectStandardInput <String>] [-RedirectStandardOutput <String>] [-WindowStyle <ProcessWindowStyle>]
- [-Wait] [-UseNewEnvironment] [<CommonParameters>]
+ [-WorkingDirectory <String>] [-LoadUserProfile] [-NoNewWindow] [-PassThru]
+ [-RedirectStandardError <String>] [-RedirectStandardInput <String>]
+ [-RedirectStandardOutput <String>] [-WindowStyle <ProcessWindowStyle>] [-Wait] [-UseNewEnvironment]
```

Comment:
Isn't `-UseNewEnvironment` for 7.x only?

Reply:
No. It exists in all versions but the behavior changed in 7.1.